multi-node/aws/pkg/config/config.go

```diff
@@ -160,8 +162,10 @@ func (c Cluster) Config() (*Config, error) {
 	config.APIServers = fmt.Sprintf("http://%s:8080", c.ControllerIP)
 	config.SecureAPIServers = fmt.Sprintf("https://%s:443", c.ControllerIP)
 	config.APIServerEndpoint = fmt.Sprintf("https://%s", c.ExternalDNSName)
-	if config.UseCalico {
-		config.K8sNetworkPlugin = "cni"
+	config.K8sNetworkPlugin = "cni"
+
+	if config.ContainerRuntime == "rkt" && config.ReleaseChannel != "alpha" {
```

Comment:
Agreed about targeting a minimum version. Will change on docs. As for determining the version in kube-aws, it looks like we can just use the  `pkg/coreosutil.GetAMIData` function and access the version from that. Then import one of the semver packages for a comparison function. I'll add in a separate commit for this to review in a bit.